app/gateway/traditional-mode.md

```diff
@@ -0,0 +1,158 @@
+<!-- vale on-->
+
+> _Figure 1: In a traditional deployment, all {{site.base_gateway}} nodes connect to the database. 
+Each node manages its own configuration._
+
+## Single node clusters
+
+A single {{site.base_gateway}} node connected to a [supported database](/gateway/configuration/#database) creates a
+{{site.base_gateway}} cluster of one node. Any changes applied via the [Admin API](/api/gateway/admin-ee/#/operations/) of this node
+will instantly take effect.
+
+For example, consider a single {{site.base_gateway}} node `A`. If we delete a previously registered Service, then any subsequent request to `A` would instantly return `404 Not Found`, as
+the node purged it from its local cache.
+
+## Multiple node clusters
+
+In a cluster of multiple {{site.base_gateway}} nodes, other nodes connected to the same database
+wouldn't be instantly notified that the Service was deleted by node `A`.  While
+the Service is **not** in the database anymore (it was deleted by node `A`), it is
+**still** in node `B`'s memory.
+
+All nodes perform a periodic background job to synchronize with changes that
+may have been triggered by other nodes. The frequency of this job can be
+configured using the [`db_update_frequency`](/gateway/configuration/#db_update_frequency) parameter in `kong.conf`.
+
+Every `db_update_frequency` seconds, all running {{site.base_gateway}} nodes will poll the
+database for any update, and will purge the relevant entities from their cache
+if necessary.
+
+If we delete a Service from node `A`, this change will not be effective in node
+`B` until node `B`'s next database poll, which will occur up to
+`db_update_frequency` seconds later (though it could happen sooner).
+
+This makes {{site.base_gateway}} clusters **eventually consistent**.
+
+## Using read-only replicas when deploying {{site.base_gateway}} clusters with PostgresSQL
+
+When using Postgres as the backend storage, you can optionally enable
+{{site.base_gateway}} to serve read queries from a separate database instance.
+
+Enabling the read-only connection support in {{site.base_gateway}}
+greatly reduces the load on the main database instance since read-only
+queries are no longer sent to it.
+
+To learn more about how to configure this feature, refer to the
+[Datastore section](/gateway/configuration/#datastore-section)
+of the {{site.base_gateway}} configuration reference.
+
+## What information is cached?
+
+For performance reasons, {{site.base_gateway}} avoids database connections when proxying
+requests, and caches the contents of your database in memory. The following entities are cached:
+
+* [Gateway Services](/gateway/entities/service/)
+* [Routes](/gateway/entities/route/)
+* [Consumers](/gateway/entities/consumer/)
+* [Plugins](/gateway/entities/plugin/)
+* Credentials, and so on 
+
+Since these values are in memory, any change made via the [Admin API](/api/gateway/admin-ee/#/operations/) of one of the nodes must be propagated to the other nodes.
```

Comment:
```suggestion
Since these values are stored in memory, any change made via the [Admin API](/api/gateway/admin-ee/#/operations/) of one of the nodes must be propagated to the other nodes.
```